hack/verify.sh

```diff
@@ -39,7 +39,7 @@ gometalinter.v2 --disable-all \
     --enable=errcheck \
     --enable=varcheck \
     --enable=goconst \
-    --enable=gas \
+    --enable=gosec \
```

Comment:
Does it mean, contributors will have to update the metalinter ?

Reply:
They don't have to.
The Travis has this problem is because it always get the latest one.
https://github.com/kubernetes-sigs/controller-runtime/blob/master/.travis.yml#L18